### DISCUSSION TITLE
Free vpn servers

### DISCUSSION THREAD

**Discussion by iliyofksy** [WeAreMahsaAmini/Core#252] [Ideas]
Server IP: 35.230.128.117
IPsec PSK: zeoKFHHtwwHgKVsqbPHf
Username: vpnuser
Password: yxYs7k8uLMWxDSkX
===========================
Server IP: 34.142.123.19
IPsec PSK: nP9KJX7cg9Wi6cTVa93J
Username: vpnuser
Password: zKT8tFa4gMK3mZiq
==========================
 Server IP: 34.77.103.196
IPsec PSK: VcoNvoejnFFxZkfNqw7g
Username: vpnuser
Password: EuN3TYNxPnE6PwR3
 ===================
 Server IP: 34.159.201.93
IPsec PSK: 8efDWkJZjriruEf9LgVj
Username: vpnuser
Password: jfnBae49X6WbZH9N
=====================
Server IP: 35.204.60.93
IPsec PSK: gSCjjEozFVr3aYMj6PVm
Username: vpnuser
Password: hzbAEFFzGZySPAZz
 ======================
 Server IP: 34.65.95.175
IPsec PSK: EDe7pGkjxVtFhvV6qdyP
Username: vpnuser
Password: aZs9duLiKTVJbtBS
=========================

**Comment by iliyofksy** [comment]
I will update these vpn servers every 24 hours bcz they will ban these ip 
Please inform others

**Reply by norsnap** [reply]
unfortunately ipsec ports (500, 4500) blocked in all ISPs, only working solution right now is shadowsocks + v2ray.
check this https://github.com/WeAreMahsaAmini/Core/discussions/62

**Reply by iliyofksy** [reply]
Thanks for update I will follow #62 and share domain names.

**Comment by sajjad5221** [comment]
Unfortunately, these servers are not working. Can you update them?
@iliyofksy
